python/pyspark/sql/tests.py

```diff
@@ -3616,6 +3616,34 @@ def test_vectorized_udf_basic(self):
                         bool_f(col('bool')))
         self.assertEquals(df.collect(), res.collect())
 
+    def test_register_vectorized_udf_basic(self):
+        from pyspark.sql.functions import pandas_udf
+        from pyspark.rdd import PythonEvalType
+        twoArgsPandasUDF = pandas_udf(lambda x: len(x), IntegerType())
```

Comment:
The name is wrong: there is only one arg.

Reply:
`twoArgsPandasUDF` -> `two_args_pandas_udf` too.